BraveWallet/Crypto/Stores/TransactionConfirmationStore.swift

```diff
@@ -181,12 +184,12 @@ public class TransactionConfirmationStore: ObservableObject {
   }
   
   func confirm(transaction: BraveWallet.TransactionInfo) {
-    txService.approveTransaction(transaction.id) { success in
+    txService.approveTransaction(.eth, txMetaId: transaction.id) { success, error, message  in
```

Comment:
Probably should get coin type from the selected account instead of hardcoding to `eth`